snakemake/deployment/conda.py

```
@@ -704,59 +695,46 @@ def _get_cmd(self, cmd):
+            locate_cmd = f"where {frontend}"
 
-        for frontend in frontends:
-            # Use type here since conda now is a function.
-            # type allows to check for both functions and regular commands.
-            if not ON_WINDOWS or shell.get_executable():
-                locate_cmd = f"type {frontend}"
+        try:
+            shell.check_output(
+                self._get_cmd(locate_cmd), stderr=subprocess.STDOUT, text=True
+            )
+        except subprocess.CalledProcessError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)